pkg/controller/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -363,6 +363,60 @@ var _ = Describe("HyperconvergedController", func() {
 				Expect(latestHCO.Status.RelatedObjects).To(ContainElement(*kubevirtRef))
 			})
 
+			It("should update resource versions of objects in relatedObjects even when there is no update on secondary CR", func() {
+
+				expected := getBasicDeployment()
+				cl := expected.initClient()
+
+				r := initReconciler(cl, nil)
+
+				// Reconcile to get all related objects under HCO's status
+				res, err := r.Reconcile(context.TODO(), request)
+				Expect(err).To(BeNil())
+				Expect(res).Should(Equal(reconcile.Result{}))
+
+				// Update Kubevirt's resource version (an example of secondary CR)
+				foundKubevirt := &kubevirtcorev1.KubeVirt{}
+				Expect(
+					cl.Get(context.TODO(),
+						types.NamespacedName{Name: expected.kv.Name, Namespace: expected.kv.Namespace},
+						foundKubevirt),
+				).To(BeNil())
```

Comment:
Won't block the PR for that, but for errors, please use `Expect(somethingThatMayCauseError).ToNot(HaveOccurred())` - for readability of the test.